### PR TITLE
Add an example of a NTC thermistor

### PR DESCRIPTION
**Pull request by bhusang**

### CODE REVIEW

File: components/murata/NCP03WF104F05RL.stanza [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+  port p : pin[[1 2]]
+
+  val sym = resistor-sym(ResistorTherm)
+  symbol = sym(p[1] => sym.p[1], p[2] => sym.p[2])
+  val pkg = ipc-two-pin-landpattern("0201")
+  landpattern = pkg(p[1] => pkg.p[1], p[2] => pkg.p[2])
+
+  val generic-props = GenericPin(min-max(0.0, 5.0), 2000.0)
+
+  property(p[1].generic-pin) = generic-props
```
Comment (d-haldane) [inline components/murata/NCP03WF104F05RL.stanza:33]: generic-pin was meant more for IC pins. I would replace this with a property associated with resistors, like 

```
rated-voltage = 5.0
```

Comment (bhusang) [reply components/murata/NCP03WF104F05RL.stanza:33]: Ok, that makes sense. I just looked for any "compatible" property that had a max-voltage associated with it.

Comment (bhusang) [reply components/murata/NCP03WF104F05RL.stanza]: BTW, the PWL macro only takes tuples of doubles, but we may want to upgrade it to accept a Tuple<Double, Toleranced> -> Toleranced? (another PR, I think)

Comment (bhusang) [reply components/murata/NCP03WF104F05RL.stanza]: Technically the `rated-voltage` property should be a relative voltage (i.e. the max voltage difference between p[1] and p[2] for rated-voltage = 5.0. But I am not sure how to code that (yet).

**Review by d-haldane — APPROVED** [review]

**Pull request merged**